# FastQC

[![Travis Build Status](https://travis-ci.org/s-andrews/FastQC.svg?branch=master)](https://travis-ci.org/s-andrews/FastQC)

FastQC is a program designed to spot potential problems in high througput sequencing datasets.  It runs a set of analyses on one or more raw sequence files in fastq or bam format and produces a report which summarises the results.

![FastQC Screenshot](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc.png)

FastQC will highlight any areas where this library looks unusual and where you should take a closer look. The program is not tied to any specific type of sequencing technique and can be used to look at libraries coming from a large number of different experiment types (Genomic Sequencing, ChIP-Seq, RNA-Seq, BS-Seq etc etc).

This project page contains the source code for the application and is only really useful only to people wanting to develop new functionality or trace bugs in FastQC.  If you just want to run the program then you want to go to the [**project web page**](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) where you can download the compiled pacakges for Windows, OSX and Linux.
