<html>

<head>
<title>3.3 - Per Sequence Quality Scores</title>
<style type="text/css">
	body {
		font-family: sans-serif;
	}
</style>
</head>
<body>
<h1>3.3 - Per Sequence Quality Scores</h1>
<h2>Summary</h2>
<p>
The per sequence quality score report allows you to see if a subset
of your sequences have universally low quality values.  It is often
the case that a subset of sequences will have universally poor quality,
often because they are poorly imaged (on the edge of the field of view
etc), however these should represent only a small percentage of the total
sequences.
</p>
<p><img src="per_sequence_quality.png"></p>
<p>
If a significant proportion of the sequences in a run have overall low
quality then this could indicate some kind of systematic problem - possibly
with just part of the run (for example one end of a flowcell).
</p>

<p>
Results from this module will not be displayed if your input is a BAM/SAM
file in which quality scores have not been recorded.
</p>


<h2>Warning</h2>
<p>
A warning is raised if the most frequently observed mean quality is below 27 -
this equates to a 0.2% error rate.
</p>

<h2>Failure</h2>
<p>
An error is raised if the most frequently observed mean quality is below 20 -
this equates to a 1% error rate.
</p>

<h2>Common reasons for warnings</h2>
<p>
This module is generally fairly robust and errors here usually indicate a general
loss of quality within a run.  For long runs this may be alleviated through quality
trimming.  If a bi-modal, or complex distribution is seen then the results should
be evaluated in concert with the per-tile qualities (if available) since this might
indicate the reason for the loss in quality of a subset of sequences.
</p>
<hr />
<ul>
	<li><a href="../">FastQC Help Top</a>
	<ul>
		<li><a href="../1%20Introduction/1.1%20What%20is%20FastQC.html">1.1 What is FastQC.html</a></li>
		<li><a href="../2%20Basic%20Operations/">2 Basic Operations</a>
			<ul>
				<li><a href="../2%20Basic%20Operations/2.1%20Opening%20a%20sequence%20file.html">2.1 Opening a sequence file</a></li>
				<li><a href="../2%20Basic%20Operations/2.2%20Evaluating%20Results.html">2.2 Evaluating Results</a></li>
				<li><a href="../2%20Basic%20Operations/2.3%20Saving%20a%20Report.html">2.3 Saving a Report</a></li>
			</ul>
		</li>
		<li><a href="../3%20Analysis%20Modules/">3 Analysis Modules</a>
			<ul>
				<li><a href="../3%20Analysis%20Modules/1%20Basic%20Statistics.html">3.1 Basic Statistics</a></li>
				<li><a href="../3%20Analysis%20Modules/2%20Per%20Base%20Sequence%20Quality.html">3.2 Per Base Sequence Quality</a></li>
				<li><a href="../3%20Analysis%20Modules/3%20Per%20Sequence%20Quality%20Scores.html">3.3 Per Sequence Quality Scores</a></li>
				<li><a href="../3%20Analysis%20Modules/4%20Per%20Base%20Sequence%20Content.html">3.4 Per Base Sequence Content</a></li>
				<li><a href="../3%20Analysis%20Modules/5%20Per%20Sequence%20GC%20Content.html">3.5 Per Sequence GC Content</a></li>
				<li><a href="../3%20Analysis%20Modules/6%20Per%20Base%20N%20Content.html">3.6 Per Base N Content</a></li>
				<li><a href="../3%20Analysis%20Modules/7%20Sequence%20Length%20Distribution.html">3.7 Sequence Length Distribution</a></li>
				<li><a href="../3%20Analysis%20Modules/8%20Duplicate%20Sequences.html">3.8 Duplicate Sequences</a></li>
				<li><a href="../3%20Analysis%20Modules/9%20Overrepresented%20Sequences.html">3.9 Overrepresented Sequences</a></li>
				<li><a href="../3%20Analysis%20Modules/10%20Adapter%20Content.html">3.10 Adapter Content</a></li>
				<li><a href="../3%20Analysis%20Modules/11%20Kmer%20Content.html">3.11 Kmer Content</a></li>
				<li><a href="../3%20Analysis%20Modules/12%20Per%20Tile%20Sequence%20Quality.html">3.12 Per Tile Sequence Quality</a></li>
			</ul>
		</li>
	</ul>
</ul>
</body>
</html>