<html>

<head>
<title>3.5 - Per Sequence GC Content</title>
<style type="text/css">
	body {
		font-family: sans-serif;
	}
</style>
</head>
<body>
<h1>3.5 - Per Sequence GC Content</h1>
<h2>Summary</h2>
<p>
This module measures the GC content across the whole length
of each sequence in a file and compares it to a modelled
normal distribution of GC content.
</p>

<p><img src="per_sequence_gc_content.png"></p>

<p>
In a normal random library you would expect to see a roughly
normal distribution of GC content where the central peak
corresponds to the overall GC content of the underlying genome.
Since we don't know the the GC content of the genome the modal
GC content is calculated from the observed data and used to
build a reference distribution.
</p>

<p>
An unusually shaped distribution could indicate a contaminated
library or some other kinds of biased subset.  A normal distribution
which is shifted indicates some systematic bias which is independent
of base position.  If there is a systematic bias which creates a
shifted normal distribution then this won't be flagged as an error
by the module since it doesn't know what your genome's GC content
should be.
</p>

<h2>Warning</h2>
<p>
A warning is raised if the sum of the deviations from the normal
distribution represents more than 15% of the reads.
</p>

<h2>Failure</h2>
<p>
This module will indicate a failure if the sum of the deviations from
the normal distribution represents more than 30% of the reads.
</p>

<h2>Common reasons for warnings</h2>
<p>
Warnings in this module usually indicate a problem with the library.  Sharp
peaks on an otherwise smooth distribution are normally the result of a specific
contaminant (adapter dimers for example), which may well be picked up by the
overrepresented sequences module.  Broader peaks may represent contamination
with a different species.
</p>
<hr />
<ul>
	<li><a href="../">FastQC Help Top</a>
	<ul>
		<li><a href="../1%20Introduction/1.1%20What%20is%20FastQC.html">1.1 What is FastQC.html</a></li>
		<li><a href="../2%20Basic%20Operations/">2 Basic Operations</a>
			<ul>
				<li><a href="../2%20Basic%20Operations/2.1%20Opening%20a%20sequence%20file.html">2.1 Opening a sequence file</a></li>
				<li><a href="../2%20Basic%20Operations/2.2%20Evaluating%20Results.html">2.2 Evaluating Results</a></li>
				<li><a href="../2%20Basic%20Operations/2.3%20Saving%20a%20Report.html">2.3 Saving a Report</a></li>
			</ul>
		</li>
		<li><a href="../3%20Analysis%20Modules/">3 Analysis Modules</a>
			<ul>
				<li><a href="../3%20Analysis%20Modules/1%20Basic%20Statistics.html">3.1 Basic Statistics</a></li>
				<li><a href="../3%20Analysis%20Modules/2%20Per%20Base%20Sequence%20Quality.html">3.2 Per Base Sequence Quality</a></li>
				<li><a href="../3%20Analysis%20Modules/3%20Per%20Sequence%20Quality%20Scores.html">3.3 Per Sequence Quality Scores</a></li>
				<li><a href="../3%20Analysis%20Modules/4%20Per%20Base%20Sequence%20Content.html">3.4 Per Base Sequence Content</a></li>
				<li><a href="../3%20Analysis%20Modules/5%20Per%20Sequence%20GC%20Content.html">3.5 Per Sequence GC Content</a></li>
				<li><a href="../3%20Analysis%20Modules/6%20Per%20Base%20N%20Content.html">3.6 Per Base N Content</a></li>
				<li><a href="../3%20Analysis%20Modules/7%20Sequence%20Length%20Distribution.html">3.7 Sequence Length Distribution</a></li>
				<li><a href="../3%20Analysis%20Modules/8%20Duplicate%20Sequences.html">3.8 Duplicate Sequences</a></li>
				<li><a href="../3%20Analysis%20Modules/9%20Overrepresented%20Sequences.html">3.9 Overrepresented Sequences</a></li>
				<li><a href="../3%20Analysis%20Modules/10%20Adapter%20Content.html">3.10 Adapter Content</a></li>
				<li><a href="../3%20Analysis%20Modules/11%20Kmer%20Content.html">3.11 Kmer Content</a></li>
				<li><a href="../3%20Analysis%20Modules/12%20Per%20Tile%20Sequence%20Quality.html">3.12 Per Tile Sequence Quality</a></li>
			</ul>
		</li>
	</ul>
</ul>
</body>
</html>