<html>

<head>
<title>3.7 - Sequence Length Distribution</title>
<style type="text/css">
	body {
		font-family: sans-serif;
	}
</style>
</head>
<body>
<h1>3.7 - Sequence Length Distribution</h1>
<h2>Summary</h2>
<p>
Some high throughput sequencers generate sequence fragments
of uniform length, but others can contain reads of wildly
varying lengths.  Even within uniform length libraries some
pipelines will trim sequences to remove poor quality base calls
from the end.
</p>
<p>
This module generates a graph showing the distribution of fragment
sizes in the file which was analysed.
</p>
<p><img src="sequence_length_distribution.png"></p>
<p>
In many cases this will produce a simple graph showing a peak only
at one size, but for variable length FastQ files this will show the
relative amounts of each different size of sequence fragment.
</p>

<h2>Warning</h2>
<p>
This module will raise a warning if all sequences are not the same length.
</p>

<h2>Failure</h2>
<p>
This module will raise an error if any of the sequences have zero length.
</p>

<h2>Common reasons for warnings</h2>
<p>
For some sequencing platforms it is entirely normal to have different read
lengths so warnings here can be ignored.
</p>
<hr />
<ul>
	<li><a href="../">FastQC Help Top</a>
	<ul>
		<li><a href="../1%20Introduction/1.1%20What%20is%20FastQC.html">1.1 What is FastQC.html</a></li>
		<li><a href="../2%20Basic%20Operations/">2 Basic Operations</a>
			<ul>
				<li><a href="../2%20Basic%20Operations/2.1%20Opening%20a%20sequence%20file.html">2.1 Opening a sequence file</a></li>
				<li><a href="../2%20Basic%20Operations/2.2%20Evaluating%20Results.html">2.2 Evaluating Results</a></li>
				<li><a href="../2%20Basic%20Operations/2.3%20Saving%20a%20Report.html">2.3 Saving a Report</a></li>
			</ul>
		</li>
		<li><a href="../3%20Analysis%20Modules/">3 Analysis Modules</a>
			<ul>
				<li><a href="../3%20Analysis%20Modules/1%20Basic%20Statistics.html">3.1 Basic Statistics</a></li>
				<li><a href="../3%20Analysis%20Modules/2%20Per%20Base%20Sequence%20Quality.html">3.2 Per Base Sequence Quality</a></li>
				<li><a href="../3%20Analysis%20Modules/3%20Per%20Sequence%20Quality%20Scores.html">3.3 Per Sequence Quality Scores</a></li>
				<li><a href="../3%20Analysis%20Modules/4%20Per%20Base%20Sequence%20Content.html">3.4 Per Base Sequence Content</a></li>
				<li><a href="../3%20Analysis%20Modules/5%20Per%20Sequence%20GC%20Content.html">3.5 Per Sequence GC Content</a></li>
				<li><a href="../3%20Analysis%20Modules/6%20Per%20Base%20N%20Content.html">3.6 Per Base N Content</a></li>
				<li><a href="../3%20Analysis%20Modules/7%20Sequence%20Length%20Distribution.html">3.7 Sequence Length Distribution</a></li>
				<li><a href="../3%20Analysis%20Modules/8%20Duplicate%20Sequences.html">3.8 Duplicate Sequences</a></li>
				<li><a href="../3%20Analysis%20Modules/9%20Overrepresented%20Sequences.html">3.9 Overrepresented Sequences</a></li>
				<li><a href="../3%20Analysis%20Modules/10%20Adapter%20Content.html">3.10 Adapter Content</a></li>
				<li><a href="../3%20Analysis%20Modules/11%20Kmer%20Content.html">3.11 Kmer Content</a></li>
				<li><a href="../3%20Analysis%20Modules/12%20Per%20Tile%20Sequence%20Quality.html">3.12 Per Tile Sequence Quality</a></li>
			</ul>
		</li>
	</ul>
</ul>
</body>
</html>