<html>

<head>
<title>3.9 - Overrepresented Sequences</title>
<style type="text/css">
	body {
		font-family: sans-serif;
	}
</style>
</head>
<body>
<h1>3.9 - Overrepresented Sequences</h1>
<h2>Summary</h2>
<p>
A normal high-throughput library will contain a diverse set
of sequences, with each individual sequence making up only a tiny
fraction of the whole.  Finding that a single sequence is very
overrepresented in the set either means that it is highly
biologically significant, or indicates that the library is
contaminated, or not as diverse as you expected.
</p>
<p>
This module lists all of the sequences which make up more than
0.1% of the total.  To conserve memory only sequences which
appear in the first 100,000 sequences are tracked
to the end of the file.  It is therefore possible that a sequence
which is overrepresented but doesn't appear at the start of the file
for some reason could be missed by this module.
</p>
<p>
For each overrepresented sequence the program will look for matches
in a database of common contaminants and will report the best hit
it finds.  Hits must be at least 20bp in length and have no more
than 1 mismatch.  Finding a hit doesn't necessarily mean that this
is the source of the contamination, but may point you in the right
direction.  It's also worth pointing out that many adapter sequences
are very similar to each other so you may get a hit reported which
isn't technically correct, but which has very similar sequence to
the actual match.
</p>
<p>
Because the duplication detection requires an exact sequence match over
the whole length of the sequence any reads over 75bp in length are truncated
to 50bp for the purposes of this analysis.  Even so, longer reads are more
likely to contain sequencing errors which will artificially increase the
observed diversity and will tend to underrepresent highly duplicated sequences.
</p>


<h2>Warning</h2>
<p>
This module will issue a warning if any sequence is found to represent
more than 0.1% of the total.
</p>

<h2>Failure</h2>
<p>
This module will issue an error if any sequence is found to represent
more than 1% of the total.
</p>

<h2>Common reasons for warnings</h2>
<p>
This module will often be triggered when used to analyse small RNA libraries
where sequences are not subjected to random fragmentation, and the same
sequence may natrually be present in a significant proportion of the library.
</p>
<hr />
<ul>
	<li><a href="../">FastQC Help Top</a>
	<ul>
		<li><a href="../1%20Introduction/1.1%20What%20is%20FastQC.html">1.1 What is FastQC.html</a></li>
		<li><a href="../2%20Basic%20Operations/">2 Basic Operations</a>
			<ul>
				<li><a href="../2%20Basic%20Operations/2.1%20Opening%20a%20sequence%20file.html">2.1 Opening a sequence file</a></li>
				<li><a href="../2%20Basic%20Operations/2.2%20Evaluating%20Results.html">2.2 Evaluating Results</a></li>
				<li><a href="../2%20Basic%20Operations/2.3%20Saving%20a%20Report.html">2.3 Saving a Report</a></li>
			</ul>
		</li>
		<li><a href="../3%20Analysis%20Modules/">3 Analysis Modules</a>
			<ul>
				<li><a href="../3%20Analysis%20Modules/1%20Basic%20Statistics.html">3.1 Basic Statistics</a></li>
				<li><a href="../3%20Analysis%20Modules/2%20Per%20Base%20Sequence%20Quality.html">3.2 Per Base Sequence Quality</a></li>
				<li><a href="../3%20Analysis%20Modules/3%20Per%20Sequence%20Quality%20Scores.html">3.3 Per Sequence Quality Scores</a></li>
				<li><a href="../3%20Analysis%20Modules/4%20Per%20Base%20Sequence%20Content.html">3.4 Per Base Sequence Content</a></li>
				<li><a href="../3%20Analysis%20Modules/5%20Per%20Sequence%20GC%20Content.html">3.5 Per Sequence GC Content</a></li>
				<li><a href="../3%20Analysis%20Modules/6%20Per%20Base%20N%20Content.html">3.6 Per Base N Content</a></li>
				<li><a href="../3%20Analysis%20Modules/7%20Sequence%20Length%20Distribution.html">3.7 Sequence Length Distribution</a></li>
				<li><a href="../3%20Analysis%20Modules/8%20Duplicate%20Sequences.html">3.8 Duplicate Sequences</a></li>
				<li><a href="../3%20Analysis%20Modules/9%20Overrepresented%20Sequences.html">3.9 Overrepresented Sequences</a></li>
				<li><a href="../3%20Analysis%20Modules/10%20Adapter%20Content.html">3.10 Adapter Content</a></li>
				<li><a href="../3%20Analysis%20Modules/11%20Kmer%20Content.html">3.11 Kmer Content</a></li>
				<li><a href="../3%20Analysis%20Modules/12%20Per%20Tile%20Sequence%20Quality.html">3.12 Per Tile Sequence Quality</a></li>
			</ul>
		</li>
	</ul>
</ul>
</body>
</html>